From eb4814f8a769829820cf97f959fded8dc6e03cef Mon Sep 17 00:00:00 2001
Message-Id: <eb4814f8a769829820cf97f959fded8dc6e03cef@dist-git>
From: John Ferlan <jferlan@redhat.com>
Date: Tue, 18 Aug 2015 14:47:14 -0400
Subject: [PATCH] api: Adjust comment for virDomainAddIOThread

https://bugzilla.redhat.com/show_bug.cgi?id=1251886

The comment for the function indicated that iothread_id had to be
a positive non-zero value; however, that wasn't checked - that is
a value of 0 is/was allowed by the API and was left up to the
hypervisor to reject the value.

More than likely this nuance was missed during the many "adjustments"
to the API in the review phase.

(cherry picked from commit 53058e11aa1d5ba9e86153ed47e97c4988b5eef6)
Signed-off-by: John Ferlan <jferlan@redhat.com>
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/libvirt-domain.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/libvirt-domain.c b/src/libvirt-domain.c
index c4377bf..55ee28d 100644
--- a/src/libvirt-domain.c
+++ b/src/libvirt-domain.c
@@ -7952,9 +7952,9 @@ virDomainPinIOThread(virDomainPtr domain,
  * @iothread_id: the specific IOThread ID value to add
  * @flags: bitwise-OR of virDomainModificationImpact
  *
- * Dynamically add an IOThread to the domain. If @iothread_id is a positive
- * non-zero value, then attempt to add the specific IOThread ID and error
- * out if the iothread id already exists.
+ * Dynamically add an IOThread to the domain. It is left up to the
+ * underlying virtual hypervisor to determine the valid range for an
+ * @iothread_id and determining whether the @iothread_id already exists.
  *
  * Note that this call can fail if the underlying virtualization hypervisor
  * does not support it or if growing the number is arbitrarily limited.
-- 
2.5.0

